corgidrp/l3_to_l4.py

```diff
@@ -313,66 +314,80 @@ def do_psf_subtraction(input_dataset, reference_star_dataset=None,
     
     return dataset_out
 
-def northup(input_dataset,correct_wcs=False):
+def northup(input_dataset,use_wcs=True):
     """
     Derotate the Image, ERR, and DQ data by the angle offset to make the FoV up to North. 
-    Now tentatively assuming 'ROLL' in the primary header incorporates all the angle offset, and the center of the FoV is the star position.
-    WCS correction is not yet implemented - TBD.
+    Now tentatively assuming the center of the FoV as the star position.
+    WCS correction is incorporated - the angle offset is calculated based on the CD information.
```

Comment:
I think we want to be clear in the docstring (Both here) and in the description of the `use_wcs` variable name that, to determine the angle we need to rotate, we either use the WCS header information (if `use_wcs` is True) or use just the `ROLL` header keyword if `use_wcs` is False (not recommended).